microservice/language/python/init/src/aggregates.py.jinja

```diff
@@ -14,9 +14,3 @@ class {{ aggregate }}(RootEntity):
 
 class {{ aggregate }}Aggregate(Aggregate[{{ aggregate }}]):
     """{{ aggregate }}Aggregate class."""
-
-    @staticmethod
-    async def create() -> UUID:
-        """Create a new instance."""
-        root = await {{ aggregate }}.create()
-        return root.uuid
```

Comment:
I think this should not be removed as it will be updated by https://github.com/minos-framework/minos-templates/pull/50

Reply:
Right.